doc/rtd/topics/debugging.rst

```diff
@@ -228,30 +253,39 @@ from **-proposed**
     apt-get install -qy cloud-init
     EOF
 
-    scp setup_proposed.sh ubuntu@$CI_VM_IP:.
-    ssh ubuntu@$CI_VM_IP -- sudo bash setup_proposed.sh
+.. code-block:: shell-session
+
+    $ scp setup_proposed.sh ubuntu@$CI_VM_IP:.
+    $ ssh ubuntu@$CI_VM_IP -- sudo bash setup_proposed.sh
 
 4. Change hostname, clean cloud-init's state, and reboot to run cloud-init
    from scratch:
 
-.. code-block:: bash
+.. code-block:: shell-session
 
-    ssh ubuntu@$CI_VM_IP -- sudo hostname something-else
-    ssh ubuntu@$CI_VM_IP -- sudo cloud-init clean --logs --reboot
+    $ ssh ubuntu@$CI_VM_IP -- sudo hostname something-else
+    $ ssh ubuntu@$CI_VM_IP -- sudo cloud-init clean --logs --reboot
 
-5. Validate **-proposed** cloud-init came up without error
+5. Validate **-proposed** cloud-init came up without error. First, we block
+   until cloud-init completes and verify from --long that the datasource is
+   the same as the one picked up from step 1. Errors will show up in --long:
 
-.. code-block:: bash
+.. code-block:: shell-script
```

Comment:
It looks like we don't have any code-block type `shell-script`, which is causing the [build to fail](https://app.travis-ci.com/github/canonical/cloud-init/jobs/590436459).

That should be I think this is supposed to be `shell-session`?

Reply:
Yes, you're absolutely right - I don't know what my brain was up to last night :joy: